src/components/sideBarContentEdit/SelectDirectory.tsx

```diff
@@ -102,22 +102,25 @@ const SelectDirectoryOption = ({
         </Flex>
       )}
       {!isDirLoading &&
-        !customPath &&
-        options.map((dir) => (
-          <Text
-            className="select-option"
-            role="button"
-            key={dir.slug}
-            onClick={() => setPath(dir.slug)}
-            color="gray.800"
-            _hover={{ bg: "blue.600", color: "gray.100" }}
-            fontSize="14px"
-            px={[2, 4, 6, 9]}
-            py={1}
-          >
-            {dir.value}
-          </Text>
-        ))}
+        options
+          .filter((dir) =>
+            dir.slug.includes(customPath ? customPath.toLowerCase() : "")
```

Comment:
Are the items in the slug always guaranteed to be in lowercase?

Reply:
yh they are